documentation/Getting started with TypeSpec specifications.md

```diff
@@ -0,0 +1,29 @@
+## What is an TypeSpec Specification?
+
+TypeSpec is a versatile language used for describing cloud service APIs and generating various API description languages, client and service code, documentation, and other related assets. It provides highly extensible core language primitives that can describe API structures common among REST, GraphQL, gRPC, and other protocols.
+
+Within the azure-rest-api-specs repository, the TypeSpec specification serves as the source to generate corresponding OpenAPI 2.0 (swagger) API documentation. By utilizing the model and operation patterns found in the standard Azure typespec-core and typespec-resource-manager libraries, TypeSpec offers a much simpler way to express service APIs and ensures compliance with the Azure REST API guidelines for swagger specifications.
+
+**Please note** TypeSpec specification is REQUIRED for all new ARM services.
+
+### TLDR: Key checklist for TypeSpec PR in azure-rest-api-spec repos
+
+#### General:
+- :white_check_mark: Do include `tspconfig.yaml`.
```

Comment:
Need the standard tspconfig.yaml for data-plane specs (it looks just like the ARM ones)

Reply:
It was mentioned and linked below in Data-plane section.